README.md

```diff
@@ -35,14 +35,26 @@ instructions](docs/installation.md).
 
```

Comment:
My comment refer to line 33, which is one line before what we see. It was not clear.